configuration/configuration.go

```diff
@@ -1,14 +1,15 @@
 package configuration
 
 import (
-	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"reflect"
 	"strings"
 	"time"
+
+	"github.com/pkg/errors"
```

Comment:
I dont believe we need this here. We can stick with the standard library `errors` package here as we do not take anything out of this package in this file sa far as I can tell.